src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Pipeline/HealthCheckTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NUnit.Framework;
+
+namespace SFA.DAS.EmployerIncentives.Api.AcceptanceTests.Pipeline
+{
+    [TestFixture]
+    [Parallelizable]
+    public class WhenCheckingPipeLine
+    {
+        [Test]
+        [Ignore("being reworked")]
+        public async Task CallPingEndpoint_ThenShouldReturnOkResponse()
+        {
+            var f = new HealthCheckFixture();
+            var r =  await f.Client.GetAsync("/ping");
+            r.StatusCode.Should().Be(HttpStatusCode.OK);
+        }
+
+        [Test]
+        [Ignore("being reworked")]
+        public async Task CallHealthEndpoint_ThenShouldReturnOkResponse()
+        {
+            var f = new HealthCheckFixture();
+            var r = await f.Client.GetAsync("/health");
+            r.StatusCode.Should().Be(HttpStatusCode.OK);
+        }
+    }
+
+    public class HealthCheckFixture
```

Comment:
No Test fixture files - These become massively bloated and confusing. Please use the SFA.DAS.Testing package for any autofixture creating and mocking.

Reply:
Can we discuss on monday?

Reply:
removed